ReactNativeClient/lib/services/CommandService.ts

```diff
@@ -75,6 +76,8 @@ export default class CommandService extends BaseService {
 	private commandPreviousStates_:CommandStates = {};
 	private mapStateToPropsIID_:any = null;
 
+	private keymapService = KeymapService.instance();
```

Comment:
That should be passed as a dependency to initialize() (like the store now). This makes it easier to control the initialization order, or indeed to find bugs in the program when we realize that for example the keymap service is initialized after the command service. Dependency injection also makes it easier to test.

Reply:
I see. I added an extra parameter for the KeymapService instance to the initialize() function. The instance is now created in the `app.js` file and passed to CommandService.